licence-header.txt

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2021 Radix Publishing Ltd incorporated in Jersey (Channel Islands).
+/* Copyright $YEAR Radix Publishing Ltd incorporated in Jersey (Channel Islands).
```

Comment:
This is not necessary. I asked @russellharvey and he said this should always be 2021.